### PR TITLE
Update to 0.6.2

### PR DESCRIPTION
**Pull request by tjof2**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by tjof2** [comment]
@conda-forge-admin, please rerender

**Comment by ericpre** [comment]
Anaconda cloud is down at the moment, so we will need to wait a bit!
https://gitter.im/conda-forge/conda-forge.github.io

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/tjof2/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 33da930fb15f1b7e7ede00f4f04eed0159612fe089d7ef7acca09457427d7a8a
+  sha256: 65158ca6b46bd3b4cf96123b513d1348df18c0277f5cc6946b227149a8b28401
   patches:
     - 0001-Remove-setup_requires.patch
```
Comment (ericpre) [inline recipe/meta.yaml]: We should be able to remove the patch `0001-Remove-setup_requires.patch`?

**Review by ericpre — COMMENTED** [review]
Anaconda cloud seems to be back!

File: recipe/meta.yaml (hunk 1 above)
Comment (tjof2) [reply recipe/meta.yaml]: Ah yes you're right.

**Comment by tjof2** [comment]
@conda-forge-admin, please rerender

**Comment by ericpre** [comment]
```
Trying to apply patch as-is
INFO:conda_build.source:Trying to apply patch as-is
INFO conda_build.source:apply_patch(598): Trying to apply patch as-is
patching file setup.py
Hunk #1 FAILED at 35.
1 out of 1 hunk FAILED -- saving rejects to file setup.py.rej
Warning: failed to download source.  If building, will try again after downloading recipe dependencies.
Error was: 
Command '['/opt/conda/bin/patch', '-p1', '--ignore-whitespace', '-i', '/home/conda/recipe_root/march.patch']' returned non-zero exit status 1.
Failed to download or patch source. Please see build log for info.
```
The march may need to updating...

**Comment by tjof2** [comment]
Ok I'll take a look later today, thanks!

**Pull request merged**